src/Core/Services/OpenAPI/OpenApiDocumentor.cs

```diff
@@ -957,9 +955,9 @@ private static OpenApiMediaType CreateResponseContainer(string responseObjectSch
         private Dictionary<string, OpenApiSchema> CreateComponentSchemas()
         {
             Dictionary<string, OpenApiSchema> schemas = new();
-
+            RuntimeConfig runtimeConfig = _runtimeConfigProvider.GetConfig();
```

Comment:
This function should mirror input arguments change that you made for `BuildPaths(...)` no need to get runtime config again in the middle of the operation

Reply:
Unresolving since this is still an issue.

Reply:
thanks had this committed but not pushed.